src/components/MovieDetails/index.tsx

```diff
@@ -133,6 +133,16 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
 
   const movieAttributes = [];
```

Comment:
```suggestion
  const movieAttributes: React.ReactNode[] = [];
```

If you mark this as an array of React nodes here, you won't need to specify it in the map below.

Reply:
Thanks!